pkg/client/scanner/scanner.go

```diff
@@ -31,32 +35,30 @@ type Config struct {
 }
 
 // Available list all available scanner tools that can be used.
-var Available = [...]Tool{
-	Licensee,
-	Scancode,
-	Composer,
-	FileHasher,
-}
+var Available []Tool
 
 // Default is the default scanner tools that shall be used if no particular tool is selected.
-var Default Tool = Licensee
+var Default Tool
 
-// Licensee represents the latest usable Licensee version
-var Licensee = Tool{
-	Name:      "Licensee",
-	Version:   "9.13.0",
-	DockerImg: "docker.pkg.github.com/osrgroup/product-model-toolkit/scanner-licensee:9.13.0",
-	Cmd:       `/bin/bash -c "licensee detect /input/ --json > /result/result.json"`,
-	Results:   []string{"result.json"},
-}
+// init identifies available scanner tools
+func init() {
+
+	// Open JSON file and read as a byte array
+	jsonFile, err := os.Open("pkg/client/scanner/scanner_tools.json")
+	if err != nil {
+		fmt.Println(err)
+	}
+	byteValue, _ := ioutil.ReadAll(jsonFile)
+	defer jsonFile.Close()
+
+	// Decode JSON data and populate list of scanner tools
+	err = json.Unmarshal(byteValue, &Available)
+	if err != nil {
+		log.Println(err)
+	}
 
-// Scancode represents the latest usable Scancode version
-var Scancode = Tool{
-	Name:      "Scancode",
-	Version:   "3.1.1",
-	DockerImg: "docker.pkg.github.com/osrgroup/product-model-toolkit/scanner-scancode:3.1.1",
-	Cmd:       `/bin/bash -c "./scancode --spdx-tv /result/result.spdx /input"`,
-	Results:   []string{"result.spdx"},
+	// Assign default scanner tool
```

Comment:
Please remove comments inside of functions. 
If a statement or some logic are to complicated extract it into a own function.